### PR TITLE
User profile

### PR DESCRIPTION
**Pull request by yp0975eu**
Feel free to have a look and make suggestions.

### CODE REVIEW

**Comment by yp0975eu** [comment]
I think we are ready for a proper review #11

File: lmn/views/views_users.py [hunk 1]
```diff
@@ -18,8 +18,23 @@ def user_profile(request, user_pk):
...
+    if request.method == 'POST':
+        form = ProfileForm(request.POST, instance=request.user.profile)
+        if form.is_valid():
+            profile_form = form.save(commit=False)
+            profile_form.user = request.user
+            profile_form.save()
+
+    user = request.user
+    usernotes = Note.objects.filter(user=user.pk).order_by('-posted_date')
+    form = ProfileForm(instance=user.profile)
```
Comment (NateBigStone) [inline lmn/views/views_users.py:30]: Should be doing a check here if user has no profile and load a blank form

File: lmn/views/views_users.py [hunk 2]
```diff
@@ -18,8 +18,23 @@ def user_profile(request, user_pk):
...
 def my_user_profile(request):
-    # TODO - editable version for logged-in user to edit their own profile
-    return redirect('user_profile', user_pk=request.user.pk)
+    if request.method == 'POST':
+        form = ProfileForm(request.POST, instance=request.user.profile)
+        if form.is_valid():
+            profile_form = form.save(commit=False)
+            profile_form.user = request.user
+            profile_form.save()
+
```
Comment (NateBigStone) [inline lmn/views/views_users.py:27]: Is there a link to this 'edit profile' page?

**Review by NateBigStone — CHANGES_REQUESTED** [review]

File: lmn/views/views_users.py (hunk 1 above)
Comment (yp0975eu) [reply lmn/views/views_users.py:30]: Every user has a profile. It does load a blank form if there is no data.
Screen Shot 2020-11-29 at 2 21 31 PM

File: lmn/views/views_users.py (hunk 2 above)
Comment (yp0975eu) [reply lmn/views/views_users.py:27]: Yes, See base.html

**Review by NateBigStone — APPROVED** [review]
👍

**Pull request merged**